app/models/news_log.rb

```diff
@@ -11,6 +11,8 @@ class NewsLog
   field :news_release_number, type: String
   field :aasm_state
   field :opa_id,  type: Integer
+  # field :imported_from_old ,  type: boolean
```

Comment:
please enable this and while importing mark them old NRL's with this flag

Reply:
Added flag for migrated data